scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -44,21 +44,21 @@ const indexMapping = {
   },
 }
 
-const archives = ['CSOR-04-05-01', 'GNDL-45-01', 'CEDW-30-16-01', 'GNDL-45-02', 'CEDW-20-02-08', 'nd-life', 'GNDL-45-04', 'CTAO-01-28', 'GNDL-45-05']
+// const archives = ['CSOR-04-05-01', 'GNDL-45-01', 'CEDW-30-16-01', 'GNDL-45-02', 'CEDW-20-02-08', 'nd-life', 'GNDL-45-04', 'CTAO-01-28', 'GNDL-45-05']
 const determineProvider = (manifest) => {
   if (manifest.id.match(/\/[0-9]{4}[.](.*)\/manifest$/)) {
     return 'Snite Museum of Art'
   }
 
-  const res = archives.find((testId) => {
-    if (manifest.id.includes(testId)) {
-      return true
-    }
-    return false
-  })
-  if (res) {
-    return 'University Archives'
-  }
+  // const res = archives.find((testId) => {
```

Comment:
Is it intentional to comment out this behavior? With source control, my preference is to remove code instead of commenting it out.

Reply:
I wanted to make sure this was no longer needed in any form before it was permanently removed.

Reply:
Could you add a TODO comment stating this intention? Then, at a future date, when its clear (or not) we can take action on tidying this up (with its own commit and its own message explaining the reason).

Reply:
I believe @jonhartzler is doing some work in this area as well, since he is working with this file as well I feel it is okay to leave this as commented out, so that he can address the change in his merge.

Reply:
that is true i have made changes to this function. Add the notes i think then I will fix the conflicts when I merge.